### DISCUSSION TITLE
[Idea] "side page" modals

### DISCUSSION THREAD

**Discussion by v23ent** [Laravel-Backpack/community-forum#1143] [Ideas]
Hi laravel-backpack community!
What would be the best way to implement "side page" like modals in Backpack ? Please advise
![photo_2024-09-03 16 41 06](https://github.com/user-attachments/assets/6341c98e-1909-48cb-82d2-166d2ad3f5c1)

**Comment by jcastroa87** [comment]
Hello @v23ent 

We currently don't have that feature, but it could be a great opportunity for you to contribute by developing an add-on. Alternatively, you can add this suggestion to our ideas list. If other users find it useful, they can upvote it, which will help us prioritize it for future updates.

Cheers.

**Reply by v23ent** [reply]
@jcastroa87 thanks for your answer!
Can you pls advise where is the ideas list ?) thx in advance!

**Reply by jcastroa87** [reply]
I updated ;)
